docs/source/getting-started/index.rst

```diff
@@ -69,12 +62,12 @@ Once the DAR file is created you will see this message in terminal ``Created .da
 Any commands starting with ``daml`` are using the :doc:`DAML Assistant </tools/assistant>`, a command line tool in the DAML SDK for building and running DAML apps.
 In order to connect the UI code to this DAML, we need to run a code generation step::
 
-    daml codegen js .daml/dist/create-daml-app-0.1.0.dar -o daml.js
+    daml codegen js .daml/dist/create-daml-app-0.1.0.dar -o ui/daml.js
 
-Now, changing to the ``ui`` folder, use Yarn to install the project dependencies::
+Now, changing to the ``ui`` folder, use ``npm`` to install the project dependencies::
 
     cd ui
-    yarn install --force --frozen-lockfile
+    npm install --force --frozen-lockfile
```

Comment:
I assume you tested that `--force --frozen-lockfile` works on npm?

Reply:
yep